app/buckets/views.js

```diff
@@ -1,9 +1,25 @@
 var api = require('../api-client');
 var bole = require('bole');
+var ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn;
 
 var log = bole('buckets-views');
 
 
+exports.list_buckets = [
+
+  ensureLoggedIn('/'),
```

Comment:
doesn't this want to be `ensureLoggedIn('/login')` now?

Reply:
Yep, quite prolly. It's wrong in a couple of other places too, which is where I copypasta'd it from. I'll correct them all and update. 👍